src/Athens-Examples/AthensSurfaceExamples.class.st

```diff
@@ -302,7 +303,9 @@ AthensSurfaceExamples class >> example4 [
 		can pathTransform translateX: 5 Y: 5.
 		can drawShape: path.
 	].
-	Display getCanvas drawImage: 	surf asForm  at: 0@0
+
+	surf asForm asMorph openInWindow. 
```

Comment:
Could it be

`surf asMorph openInWindow`?

Or do we want that in general surfaces can be transformed into forms?